### PR TITLE
Remove usage of class param

### PR DESCRIPTION
**Pull request by Ayushi-12**
### Summary
Remove usage of class parameter
### Type
Breaking change

### CODE REVIEW

File: praetorian_cli/handlers/add.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def definition(controller, path, name):
     if name is None:
         name = path.split('/')[-1]
     try:
-        controller.upload(path, "definition", f"definitions/{name}")
+        controller.upload(path)
```
Comment (peter-kwan) [inline praetorian_cli/handlers/add.py:57]: I think we do need to have "definitions/" as a prefix for these files. The UI depends on this naming convention to load the proofs of exploit.

**Review by peter-kwan — CHANGES_REQUESTED** [review]

File: praetorian_cli/handlers/add.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def upload(controller, path, name):
     PATH : File path in the local system
     """
     try:
-        controller.upload(path, "manual", name)
+        controller.upload(path)
```
Comment (Ayushi-12) [inline praetorian_cli/handlers/add.py:39]:
```suggestion
        controller.upload(path, name)
```

File: praetorian_cli/handlers/add.py (hunk 1 above)
Comment (Ayushi-12) [reply praetorian_cli/handlers/add.py:57]:
```suggestion
        controller.upload(path, f"definitions/{name}")
```

File: praetorian_cli/plugins/commands/report.py [hunk 3]
```diff
@@ -50,15 +50,14 @@ def workflow(self):
 
         if click.prompt(f'Upload {path} finding to Chariot? (RECOMMENDED)',
                         type=bool, default=True):
-            self.controller.upload(path, f"definitions/{sow}/{risk_name}")
+            self.controller.upload(path)
```
Comment (Ayushi-12) [inline praetorian_cli/plugins/commands/report.py:53]:
```suggestion
            self.controller.upload(path, f"definitions/{sow}/{risk_name}")
```

File: praetorian_cli/plugins/commands/report.py [hunk 4]
```diff
@@ -50,15 +50,14 @@ def workflow(self):
...
 
         while click.prompt(
                 'Upload any additional engagement files to Chariot', type=bool, default=False):
             path = fzf_file(click.prompt('Enter glob pattern to search for files',
                                          type=str, default='./**/*'))
             if click.prompt(f'Upload {path}', type=bool, default=True):
-                self.controller.upload(
-                    path, f"files/{sow}/{os.path.basename(path)}")
+                self.controller.upload(path)
```
Comment (Ayushi-12) [inline praetorian_cli/plugins/commands/report.py:60]:
```suggestion
                self.controller.upload(
                    path, f"files/{sow}/{os.path.basename(path)}")
```

**Review by Ayushi-12 — COMMENTED** [review]
Fix name parameter

File: praetorian_cli/sdk/chariot.py [hunk 5]
```diff
@@ -84,16 +84,16 @@ def unlink(self, username: str, id: str = ""):
         return resp.json()
 
     @verify_credentials
-    def upload(self, name: str, clss: str, upload_path: str = ""):
+    def upload(self, name: str, upload_path: str = ""):
         path = upload_path if upload_path else name
         with open(name, 'rb') as content:
-            self._upload(path, clss, content)
+            self._upload(path, )
```
Comment (Ayushi-12) [inline praetorian_cli/sdk/chariot.py:90]:
```suggestion
            self._upload(path, content)
```

**Review by peter-kwan — APPROVED** [review]

**Pull request merged**